Makefile

```diff
@@ -6,6 +6,15 @@ MACHINE = $(shell uname -m)
 BUILD_IMAGE = golang:1.13.5
 BASE_IMAGE = storageos/base-image:0.2.1
 
+# Set the new version before running the release target.
+NEW_VERSION = v2.0.0
+
+# Variables to be used when running the make targets in a container.
+CACHE_DIR = $(shell pwd)/.cache
+PROJECT = github.com/storageos/cluster-operator
+GOARCH ?= amd64
+GO_VERSION = 1.14.2
```

Comment:
Will unify these with the build image variables a separate PR. This is for running all the tooling in container for now.